itest/lnd_payment_test.go

```diff
@@ -256,6 +256,14 @@ func testListPayments(ht *lntest.HarnessTest) {
 				endDate:  createTimeSeconds - 1,
 				expected: false,
 			},
+			{
```

Comment:
think we can remove the line `modified:   itest/lnd_payment_test.go` in the commit msg

Reply:
done 07bf0a1968280138fbc5496cac3774bdbb89e82e